### ISSUE TITLE
Boot device selection not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#26]
```
What steps will reproduce the problem?
1. Select any device to boot from other than default and hard drive
2. Will automatically boot to hard drive, unless it is not mounted.
3.

What is the expected output? What do you see instead?
Should boot to CD/DVD or Floppy automatically

What version of the product are you using? On what operating system?
0.9.5 on Nexus 4

Please provide any additional information below.


```

Original issue reported on code.google.com by `dabb...@gmail.com` on 21 Jan 2013 at 7:37

**Comment by GoogleCodeExporter** [comment]
```
Also, it worked on previous releases.
```

Original comment by `dabb...@gmail.com` on 21 Jan 2013 at 7:38

**Comment by GoogleCodeExporter** [comment]
```
Same problem on Samsung Galaxy 3 with limbo 0.9.5

```

Original comment by `chrisdes...@gmail.com` on 23 Jan 2013 at 3:52

**Comment by GoogleCodeExporter** [comment]
```
Boot param was broken during 0.9.5. Tentatively scheduling fix for 0.9.6.
```

Original comment by `ideal2i...@gmail.com` on 23 Jan 2013 at 6:59
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Rel 0.9.6
```

Original comment by `ideal2i...@gmail.com` on 2 Feb 2013 at 4:40
- Changed state: **Done**
